src/Frontend/Core/Engine/TemplateModifiers.php

```diff
@@ -11,6 +11,7 @@
 
 use Frontend\Core\Engine\Model as FrontendModel;
 use Frontend\Core\Engine\Block\Widget as FrontendBlockWidget;
+use Frontend\Core\Language\Language;
```

Comment:
doesn't this give conflcts because the class name is already taken in this namespace?